recipes/pynvml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pynvml" %}
+{% set build = 0 %}
```

Comment:
I think you could avoid the definition of this env variable. Instead, I recommend to use only the build `number` field, just like `number: 0`. When a new release will be available, a bot will detect new versions of the package, and I don't know if setting build number like this could mess with the bot.